src/participant/participant.ts

```diff
@@ -1160,10 +1197,19 @@ export default class ParticipantController {
       ...(sampleDocuments ? { sampleDocuments } : {}),
     });
 
-    await this.streamChatResponseContentWithCodeActions({
-      modelInput,
-      stream,
-      token,
+    const { hasCodeBlock, outputLength } =
+      await this.streamChatResponseContentWithCodeActions({
+        modelInput,
+        stream,
+        token,
+      });
+
+    this._telemetryService.trackCopilotParticipantResponse({
+      command: 'query',
+      has_cta: false,
```

Comment:
Should this be true when we have a code block? There's the open in playground and run actions.

Reply:
True, but then it seems like `has_runnable_content` and `has_cta` would be duplicated quite often. When reading the design, I interpreted `has_runnable_content` to be the flag indicating if we have a code block, while `has_cta` would contain any other cta content - such as docs links/open schema button. I don't have a strong opinion and will let you, @alenakhineika and @GaurabAryal to make the call.

Reply:
Makes sense, I'm cool with anything here, thinking about it more it won't end up being confusing or losing any data whichever way we end up going so I'll resolve.